src/config.ts

```diff
@@ -30,4 +32,20 @@ export const config: AppConfig = {
 	repo: {
 		project: 'Automattic/wp-calypso',
 	},
+
+	envs: [ 'calypso', 'jetpack' ],
 };
+
+export function envContainerConfig( environment: RunEnv ): Dockerode.ContainerCreateOptions {
+	switch ( environment ) {
+		case 'calypso':
+		default:
+			return {
+				Env: [ 'NODE_ENV=wpcalypso', 'CALYPSO_ENV=wpcalypso' ],
+			};
+		case 'jetpack':
+			return {
+				Env: [ 'NODE_ENV=jetpack-cloud-stage', 'CALYPSO_ENV=jetpack-cloud-stage' ],
```

Comment:
Should we create our own `jetpack-cloud-wpcalypso` config file for this environment? 

I don't think this is a blocker for us. This change can be done after we have created one and deploy it :)